src/graph/validator/GoValidator.cpp

```diff
@@ -69,6 +69,23 @@ Status GoValidator::validateWhere(WhereClause* where) {
   }
 
   auto expr = where->filter();
+  auto filterExprs = ExpressionUtils::collectAll(
+      expr, {Expression::Kind::kVertex, Expression::Kind::kDstProperty});
+  for (auto filterExpr : filterExprs) {
+    if (filterExpr->kind() == Expression::Kind::kDstProperty) {
+      goCtx_->joinDst = true;
+    } else {
+      auto& name = static_cast<const VertexExpression*>(filterExpr)->name();
+      if (name == "VERTEX") {
+        return Status::SemanticError("`%s' is not support in go sentence.",
+                                     filterExpr->toString().c_str());
+      }
+      if (name == "$$") {
```

Comment:
How about `properties($$).age` case?

Reply:
what is checked is the extracted expression

Reply:
Please add the test case about this.

Reply:
$$ in properties($$).age is a vertexExpression

Reply:
Ok, I missed that.